wpimath/src/main/java/edu/wpi/first/math/estimator/KalmanFilter.java

```diff
@@ -37,12 +37,22 @@
 
   /** The steady-state Kalman gain matrix. */
   @SuppressWarnings("MemberName")
-  private final Matrix<States, Outputs> m_K;
+  private Matrix<States, Outputs> m_K;
 
   /** The state estimate. */
   @SuppressWarnings("MemberName")
   private Matrix<States, N1> m_xHat;
 
+  @SuppressWarnings("MemberName")
+  private Matrix<States, States> m_P;
+
+  @SuppressWarnings("MemberName")
+  private final Matrix<Outputs, States> m_C;
```

Comment:
`final var C = m_plant.getC();` should be used instead in the relevant function.